### PR TITLE
Fix CI analyze

### PR DESCRIPTION
**Pull request by MalcolmMielle**
Trying to fix the linting CI... :(

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,6 +34,12 @@ jobs:
           channel: 'stable'
       - run: dart --version
       - run: flutter --version
+      - name:
+        run: |
+        cd example
```
Comment (prateekmedia) [inline .github/workflows/ci.yml:39]: It should be tabbed inside once. Like below:
```
        run: |
          cd example
```

Comment (MalcolmMielle) [reply .github/workflows/ci.yml]: Thanks!

**Comment by prateekmedia** [comment]
I think a better CI would be Just:
```
flutter pub get
flutter analyze . example
```

**Comment by MalcolmMielle** [comment]
> I think a better CI would be Just:
> 
> ```yaml
> flutter pub get
> flutter analyze . example
> ```

Maybe. For the linter, I need to do flutter pub get in both the main folder, and in the example folder. Would that work with your proposal ?

Btw, @prateekmedia, it seems that the exemple app is not building on macos because of bitsdojo_windows. Any idea why? I have zero knowledge of macos :S.

**Comment by prateekmedia** [comment]
> @prateekmedia, it seems that the example app is not building on macos because of bitsdojo_windows. Any idea why? I have zero knowledge of macos :S.

https://pub.dev/packages/bitsdojo_window has some instructions to follow for macos build.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -34,7 +34,12 @@ jobs:
           channel: 'stable'
       - run: dart --version
       - run: flutter --version
-      - run: flutter analyze
+      - name: analyze
+        run: |
+          cd example
+          flutter pub get
```
Comment (prateekmedia) [inline .github/workflows/ci.yml:40]: In my pc, running flutter pub get runs it for both libadwaita and example so I think cd is not needed, global `flutter pub get` would work too.

Comment (MalcolmMielle) [reply .github/workflows/ci.yml:40]: On my machine I have the same behavior, but the CI seems to behave differently. You can check one of the earlier failed attempt :(

Comment (prateekmedia) [reply .github/workflows/ci.yml:40]: Ok 👍

Comment (MalcolmMielle) [reply .github/workflows/ci.yml:40]: I agree it's sad :(

**Comment by MalcolmMielle** [comment]
I ended up removing the macos lint example. It can be trivially added back (and I think in the long run it should be since we claim to work on all platform) but I don't have the hardware and time to fix that :|.

**Comment by prateekmedia** [comment]
It can be fixed by following these steps
- Running flutter config enable macos desktop
- Now running flutter create . --platform=macos
- Now modifying the files as suggested by bitsdojo readme
- Now remove all other files from macos folder other than that we modified
- Now push the changes to the branch

Although I will do it if you dont want that.

**Comment by MalcolmMielle** [comment]
I owuld be great if you could add it. Thank you :D

**Pull request merged**